v-next/hardhat-errors/src/descriptors.ts

```diff
@@ -492,5 +492,12 @@ Please double check your script's path.`,
 
 Please check Hardhat's output for more details.`,
     },
+    TEST_TASK_ESM_TESTS_RUN_TWICE: {
+      number: 602,
+      messageTemplate: `Your project uses ESM and you've programmatically run your tests twice. This is not supported yet.`,
+      websiteTitle: "Running tests twice in an ESM project",
+      websiteDescription:
+        'You have run your tests twice programmatically and your project is an ESM project (you have `"type": "module"` in your `package.json`, or some of your files have the `.mjs` extension). This is not supported by Mocha yet.',
```

Comment:
Is there a Mocha issue for that that we could link to?

Reply:
That'd be cool to add to the websiteDescription

Reply:
This was copied directly from hardhat v2. If an issue exists, there wasn't one linked there

Reply:
I've just found one! I think it might be it - https://github.com/mochajs/mocha/issues/2706